### ISSUE TITLE
Publish `debug-log` package

### ISSUE THREAD

**Issue by black7375** [mincho-js/mincho#38]
## Description

Package and release the internal tool [`debug-log`](https://github.com/mincho-js/mincho/tree/main/packages/debug-log).

- It's a very simple, but useful
- Identify and address potential issues before we ship main package

**Comment by Jeong-jj** [comment]
Hi, when i publish `Debug-log` package i found some problems

### 1. project name


![Pasted image 20240722020954](https://github.com/user-attachments/assets/2d93c2a0-d55e-45e8-9904-eed53301ae4a)

- `mincho`에서 `mincho-js`로 project name 전면 수정
	- 새로 install & build 진행
		- install 다시 안 하면  이름 안 맞는 그대로 상태임

**Comment by Jeong-jj** [comment]
Hi I found some problems when I publish `Debug-log` package,

## 1. project name

`yarn install` & `yarn build` proceeding was sccessfully finished, But after proceeding to `yarn workspace @mincho-js/debug-log run publish`, I got the following error

![publish failed](https://github.com/user-attachments/assets/a013313f-9829-47b0-910e-d46642ef4f7f)

So I changed project name `mincho` to `mincho-js`
- After this publishing succeeded

<br/>

## 2. Version matching about `"pretify-deep-diff"`

When we proceed `yarn add @mincho-js/debug-log`, we have to choose version of "pretify-deep-diff"

![select version during install](https://github.com/user-attachments/assets/639ee27a-5256-4396-a9ca-9ad159dee4dd)

<br/>

## 3. CommonJS's `require()`

when I test `const debugLogs = require("@mincho-js/debug-log")` on `node terminal`

```bash
Uncaught:
Error [ERR_REQUIRE_ESM]: require() of ES Module D:\project_test\node_modules\chalk\source\index.js from D:\project_test\node_modules\@mincho-js\debug-log\dist\cjs\index.cjs not supported.
Instead change the require of index.js in D:\project_test\node_modules\@mincho-js\debug-log\dist\cjs\index.cjs to a dynamic import() which is available in all CommonJS modules.
    at Module.<anonymous> (D:\project_test\node_modules\@mincho-js\debug-log\dist\cjs\index.cjs:1:89) {
  code: 'ERR_REQUIRE_ESM'
}
```

<br/>

## 4. Importing JSON modules

I got a warning message about importing JSON modules

```bash
(node:30428) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
```

It could find when we use debug-log tools

<br/>

---

## Summary

- The first problem is being solved in #42 
- The rest of the problems still remain

**Comment by black7375** [comment]
Now published.
We only support ESM package.
